apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
```

Comment:
Should it be "App V2" ? somehow we have to make it clear this is a V2 only feature.